docker/package/scripts/update-binaries-list.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import os
+import sys
+import json
+
+binaries_json_path_suffix = "/tests/binaries.json"
+binaries_list_path_suffix = "/tmp/binaries.txt"
+
+
+def update_binaries(binaries, field):
+    binaries_json_path = os.environ["PWD"] + binaries_json_path_suffix
```

Comment:
nitpick: using `os.path.join` will be a bit more proper there (and in other places)